core/blockchain.go

```diff
@@ -239,18 +239,20 @@ func (bc *BlockChain) loadLastState() error {
 	bc.currentFastBlock = bc.currentBlock
```

Comment:
bc.currentFastBlock.Store(bc.CurrentBlock())

Reply:
Oh, nice catch!

Reply:
It was `go vet`, not me https://travis-ci.org/ethereum/go-ethereum/jobs/345187923#L2266.